app.rb

```diff
@@ -0,0 +1,78 @@
+require './book'
+require './student'
+require './teacher'
+require './person'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def begin_console
+    i = 0
+    loop do
+      list_options
+      i = gets.chomp
+      parameter_option(i)
+      break unless i != '7'
+    end
+  end
+
+  def create_book(title, author)
+    @books << Book.new(title, author)
+  end
+
+  def list_books()
+    puts 'Books shelf is empty.' if @books.empty?
+    @books.each { |book| puts "Title: \"#{book.title}\" Author: #{book.author}" }
+  end
+
+  def create_person(checker, age, name, parent_permission)
+    @people << if checker == 1
+                 Student.new(checker, age, parent_permission, name)
+               else
+                 Teacher.new(parent_permission, age, name)
+               end
+  end
```

Comment:
- [ ] When I create a teacher through the _**UI**_, and I list people, I notice that the _**age**_ variable is saved under the _**name**_ and vice versa. This is because you are not saving the variables the same way you declared the variables in the `initialize` method of the `Teacher` class. I would advise you to ensure that they are saved the correct way. 👍 


![image](https://user-images.githubusercontent.com/77106962/224130714-74748fb8-99ce-4b06-b2aa-343c348b5e84.png)

Reply:
Hello! 
From my end, I am not experiencing the same problem. It difficult to solve a problem I am not expiriencing.
<img width="533" alt="teach" src="https://user-images.githubusercontent.com/65546920/224229417-519b5ac8-486f-4643-94d8-a739227ea9bb.png">